kolibri/plugins/user_profile/tasks.py

```diff
@@ -53,6 +53,9 @@ def create_remote_user(self, data):
             "password": data["password"],
             "facility": facility,
         }
+        for f in ["gender", "birth_year", "id_number", "full_name"]:
+            if getattr(data["local_user_id"], f, None):
+                user_data[f] = getattr(data["local_user_id"], f, None)
```

Comment:
Should we also exclude the NOT_SPECIFIED value here too? https://github.com/learningequality/kolibri/blob/develop/kolibri/core/auth/constants/demographics.py#L5

Reply:
yep, good catch.
Changed!